test/cli/packager-layers/test.out

```diff
@@ -2,7 +2,23 @@ __package.rb:5: Argument to `layer` must be one of: `library` or `application` h
      5 |  layer 'fake'
                 ^^^^^^
 Errors: 1
-No errors! Great job.
+__package.rb:3: Unable to resolve constant `PackageSpec` https://srb.help/5002
+     3 |class Project::Root < PackageSpec
+                              ^^^^^^^^^^^
+  Did you mean `Sorbet::Private::Static::PackageSpec`? Use `-a` to autocorrect
+    __package.rb:3: Replace with `Sorbet::Private::Static::PackageSpec`
+     3 |class Project::Root < PackageSpec
+                              ^^^^^^^^^^^
+    ???: `Sorbet::Private::Static::PackageSpec` defined here
```

Comment:
Should we define this in one of the payload rbis so that it has a loc?

Reply:
Sure, I can queue that when I land this stack.